docusaurus/docs/dev-docs/api/plugins/admin-panel-api.md

```diff
@@ -233,18 +193,14 @@ The Settings API allows:
 * [creating a new setting section](#createsettingsection)
 * adding [a single link](#addsettingslink) or [multiple links at once](#addsettingslinks) to existing settings sections
 
-:::note
-Adding a new section happens in the [register](#register) lifecycle while adding links happens during the [bootstrap](#bootstrap) lifecycle.
```

Comment:
Why was it removed? Is it because it's wrong?
If correct, I think it's important to keep it as this part might appear unintuitive to users.

Reply:
But if you think it is visually too distracting, we can tone it down using an `:::info` (shades of grey) callout instead of a `:::note` one (which is blue). The info callout was unavailable in VuePress, that is when this documentation was created initially.